startup.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
I always put the first command in a bash script to be `set -eu -o pipefail`, this is defensive programming.

Reply:
@drivera-armedia Any comment on this?

Reply:
For this script it's not really relevant as all it does is sanitize some variables and set others, so there's no "commands" really that can fail, and the final command is an exec so if that fails, we're still crashing. But I have no objection to this. I'll test it and if it works fine, then I'll leave it in.